modules/utils.sh

```diff
@@ -19,11 +19,11 @@ check_result() {
     fi
 }
 
-check_user() {
-    if [ "$(id -u)" -ne 0 ]; then
-        error_msg "This command must be run as root (try using sudo)"
-        exit 2
-    fi
+call_if_defined() {
+    local command="$1"
+
+    type -t "$command" >/dev/null || return 0
```

Comment:
I'd make sure type -t is returning "function", or else if someone puts a $service_check_support script in $PATH, or shell alias, or just a file in the current directory, we will call out to that.

Reply:
This is actually intentional, so it's possible to use `call_if_defined` to run a command only if it exists (whithout having to run `which`). Note that a function takes precedence over a command in path.

Reply:
Ok, I'm a bit worried about outside manipulation of what we run. I tried setting an alias for fips_check_support,  because an alias takes precedence to a function:
```
# fips_check_support() {
> echo function busted
> }

# type -t fips_check_support
function

# alias fips_check_support="echo alias busted"

# type -t fips_check_support
alias

# fips_check_support
alias busted
```
But neither get "injected" into the script.